src/components/message.js

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { Link } from "gatsby"
+
+const Message = ({ type, text, onConfirm, onCancel}) => {
+  return (
+    <>
+      {type === "error" && (
+        <p className="w-full px-3 py-4 mb-3 text-center text-white break-words bg-red-600 rounded-lg">{text}  </p>
```

Comment:
Tons of code duplication. Don't put a new `<p>` tag for every different type, just conditionally add or remove a singular class. Other parts of Heartwood has this functionality if you want to copy it.